_extras/guide.md

```diff
@@ -67,6 +72,9 @@ Instructors should, however, walk learners through some basic operations:
 *   Create a new notebook.
 *   Enter code or data in a cell and execute it.
 *   Explain the difference between `In[#]` and `Out[#]`.
+*   Explain that the cells are independent blocks 
+and if we make a change at a top cell we need to run all the subsequent
+cells to make sure the change is reflected across the script.
```

Comment:
I think this is a concept worth including, as it often comes up in workshops. I do wonder, though, if teaching it at the very beginning of a lesson might start to overload learners and be confusing to see before people new to programming have seen much code. If people want to include it, I suggest possibly rewriting it more as an operation:
```
* Show that modifying code in an earlier cell requires re-running any subsequent
cell in order for it to reflect the change.
```

Reply:
Much better wording, thanks! I think you point out to an interesting concern: perhaps this guide can be eventually restructured so that it roughly explains which points correspond to which portions of the curriculum because some particular issues like this one can be confusing at the beginning, but they are the things that better be provided to the students at some point in a curriculum with such scope with fairly complex workflows. To keep things relatively simple, we could flag some sentences in the following way:

```suggestion
* After few episodes show that modifying code in an earlier cell requires re-running any subsequent cells in order for it to reflect the change.
```

Reply:
I don't think it needs to be a few episodes in, maybe leaving it open-ended is ok, allowing the instructor to teach it when there are adequate cells for the concept. We should be sure to match plural (cells/them) though:
```
* Show that modifying code in an earlier cell requires re-running any
subsequent cells in order for them to reflect the change.
```